test/e2e/framework/kubelet_stats.go

```diff
@@ -598,6 +598,9 @@ func (r *resourceCollector) collectStats(oldStatsMap map[string]*stats.Container
 		}
 
 		if oldStats, ok := oldStatsMap[name]; ok {
+			if oldStats.CPU == nil || cStats.CPU == nil {
```

Comment:
cStats.Memory looks like it might be subject to the same problem if that comes back nil.

Reply:
Thanks for the review... Updated the PR :)